app/assets/src/components/views/samples/ProjectUploadMenu.jsx

```diff
@@ -1,8 +1,11 @@
 import React from "react";
-import cs from "./project_upload_menu.scss";
-import BareDropdown from "~ui/controls/dropdowns/BareDropdown";
 import PropTypes from "prop-types";
+
+import BareDropdown from "~ui/controls/dropdowns/BareDropdown";
+import { logAnalyticsEvent } from "~/api/analytics";
+
```

Comment:
nit: remove empty line

Reply:
I believe our convention was to have empty lines between the import groups.